nservicebus/msmq/transportconfig.md

```diff
@@ -25,7 +25,7 @@ Because of historic reasons, the configuration for MSMQ transport has been coupl
 
 ### Queue permissions
 
-By default, queues are created with `Everyone` and `Anonymous Logon` permissions to allow messages to be sent and received without additional configuration. Whilst initially convenient, this unrestricted access may be a security concern. If your organization requires it, you should set appropriate permissions on a queue after its creation.
+By default, queues are created with `Everyone` and `Anonymous Logon` permissions to allow messages to be sent and received without additional configuration. Whilst initially convenient, this unrestricted access may be a security concern. If required, the appropriate permissions can be set on a queue after its creation.
```

Comment:
>  Whilst initially convenient, this unrestricted access may be a security concern.

Does this sentence add value? If the reader believes the previous sentence to be a security concern then the following sentence helps them fix it. If the reader did not think it is of concern, saying it might be doesn't really help them make a decision.